src/settings.rs

```diff
@@ -114,6 +121,7 @@ impl Settings {
         s.set_default("database_pool_connection_timeout", Some(30))?;
         s.set_default("database_use_test_transactions", false)?;
         s.set_default("master_secret", "")?;
+        s.set_default::<Option<String>>("database_pool_max_size", Some("10".to_owned()))?;
```

Comment:
This can be simplified to just `Some(10)`, rather than letting things auto-parse. There was some old debugging that is the source. I do note that we have `database_pool_max_size` set, but weren't including it as an option?

Reply:
We just didn't fill in a default here, so it defaults to None. The different db backends do their own default when it's None -- hence suggesting we kill it because it isn't really needed but it also doesn't match the `Default` impl setting it to None (this method's defaults values setup here and the `Default` impl values should really match and hopefully one day we won't need both versions of setting defaults #253)

Reply:
I'll add a comment in that respect so it's clearer. Thanks!

Reply:
Heh, although I see the value is 10 for both mysql and spanner.